### PR TITLE
Adding ability to remove backfilled data for a specific county

### PR DESCRIPTION
**Pull request by ghop02**
Yesterday, Dallas County reported 5k backfilled tests.  This PR adds some logic in to allow dropping a specific county's backfill from the overall totals.

### CODE REVIEW

File: scripts/update_nytimes_data.py [hunk 1]
```diff
@@ -96,6 +103,31 @@ def remove_backfilled_cases(
     return data
 
 
+def remove_county_backfilled_cases(
+    data: pd.DataFrame, backfilled_cases: List[Tuple[str, str, int]]
+) -> pd.DataFrame:
+    """Removes reported county backfilled cases from case totals.
+
+    Args:
+        data: Data
```
Comment (TomGoBravo) [inline scripts/update_nytimes_data.py:112]: Ha ha!

File: scripts/update_nytimes_data.py [hunk 2]
```diff
@@ -96,6 +103,31 @@ def remove_backfilled_cases(
...
+
+    Returns: Updated data frame.
+    """
+    for county_fips, date, cases in backfilled_cases:
+        is_on_or_after_date = data[CommonFields.DATE] >= date
+        is_fips_data_after_date = is_on_or_after_date & (data[CommonFields.FIPS] == county_fips)
+        data.loc[is_fips_data_after_date, CommonFields.CASES] -= int(cases)
+
+        # Remove county count from state counts as well
+        state_fips = county_fips[:2]
```
Comment (TomGoBravo) [inline scripts/update_nytimes_data.py:123]: factor out county fips -> state fips as function for easier changing in the future

File: scripts/update_nytimes_data.py [hunk 3]
```diff
@@ -96,6 +103,31 @@ def remove_backfilled_cases(
...
+        is_fips_data_after_date = is_on_or_after_date & (data[CommonFields.FIPS] == state_fips)
+        if is_fips_data_after_date.any():
+            data.loc[is_fips_data_after_date, CommonFields.CASES] -= cases
```
Comment (TomGoBravo) [inline scripts/update_nytimes_data.py:126]: why int(cases) above and just cases here?

File: test/update_nytimes_test.py [hunk 4]
```diff
@@ -62,6 +62,37 @@ def test_remove_ct_cases(is_ct_county):
...
+
+    backfill = [("48113", "2020-08-17", 500)]
+    data_buf = io.StringIO(
+        "fips,state,date,aggregate_level,cases\n"
+        f"48113,TX,2020-08-16,county,1000\n"
+        f"48113,TX,2020-08-17,county,1600\n"
+        f"48113,TX,2020-08-18,county,1700\n"
+        f"48,TX,2020-08-16,state,2600\n"
+        f"48,TX,2020-08-17,state,3600\n"
+        f"48,TX,2020-08-18,state,4700\n"
```
Comment (TomGoBravo) [inline test/update_nytimes_test.py:75]: add another row to show that other FIPS are not modified

File: scripts/update_nytimes_data.py (hunk 3 above)
Comment (ghop02) [reply scripts/update_nytimes_data.py:126]: ah, this was an artifact of copied code - will remove

**Pull request merged**